bin/studio-common

```diff
@@ -28,8 +28,8 @@ DOC
   exit 1
 fi
 
-# This value is automatically updated by Expeditor to reflect .hab-version
-export SUPPORTED_HAB_VERSION="0.58.0"
+SUPPORTED_HAB_VERSION=$(curl -sL https://raw.githubusercontent.com/chef/ci-studio-common/master/.hab-version | xargs echo -n)
```

Comment:
I don't think there's any need for the `| xargs echo -n`. If it's to strip the newline, assignments in `bash` already do so:
```
$ SUPPORTED_HAB_VERSION=$(curl -sL https://raw.githubusercontent.com/chef/ci-studio-common/master/.hab-version | xargs echo -n)
$ SUPPORTED_HAB_VERSION2=$(curl -sL https://raw.githubusercontent.com/chef/ci-studio-common/master/.hab-version)
$ test "$SUPPORTED_HAB_VERSION2" = "$SUPPORTED_HAB_VERSION"
$ echo $?
0
```